examples/yoti_example_flask/app.py

```diff
@@ -44,4 +45,4 @@ def auth():
 
 
 if __name__ == '__main__':
-    app.run(debug=True)
+    app.run(host="0.0.0.0", debug=True)
```

Comment:
This change was applied so that the docker app could be accessed by the host browser. I tested that this still works when not using Docker too.